Samples/Client/WebClient/index.html

```diff
@@ -156,10 +325,18 @@
                 console.log("Create answer error:", error);
             }, mediaConstraints); // type error  ); //}, null          
         }
+        else if (data.search("answer") != -1) {
+            console.log("Got answer ", dataJson);
+			pc.setRemoteDescription(new RTCSessionDescription(dataJson), onRemoteSdpSucces, onRemoteSdpError);
+        }
         else {
             console.log("Adding ICE candiate ", dataJson);
             var candidate = new RTCIceCandidate({ sdpMLineIndex: dataJson.sdpMLineIndex, candidate: dataJson.candidate });
-            pc.addIceCandidate(candidate, aic_success_cb, aic_failure_cb);
+			pc.addIceCandidate(candidate).then(_=>{
```

Comment:
nit: instead of capturing into `_` can we do `() => { /* logic */ }` ?

Reply:
sure